subsys/net/lib/nrf_provisioning/Kconfig

```
@@ -73,6 +73,14 @@ config NRF_PROVISIONING_JWT_MAX_VALID_TIME_S
 	range 0 360
 	default 300
 
+config NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN
+	bool "Display attestation token when unauthorized"
+	default y if !NRF_PROVISIONING_SHELL
```

```suggestion
	default y if !NRF_PROVISIONING_SHELL
	depends on NRF_PROVISIONING_LOG_LEVEL_INF
```

prefer to use LOG_INF instead of printk, but i'll leave that up to the code owners.